### PR TITLE
default publishInstallersandchecksums to true

### PR DESCRIPTION
**Pull request by epananth**
Issue -> https://github.com/dotnet/core-eng/issues/10655

Defaulting this true for now.

### CODE REVIEW

File: src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs [hunk 1]
```diff
@@ -255,7 +255,7 @@ private async Task<int> PromoteBuildAsync(Build build, List<Channel> targetChann
                 { "SigningValidationAdditionalParameters", _options.SigningValidationAdditionalParameters },
                 { "EnableNugetValidation", _options.DoNuGetValidation.ToString() },
                 { "EnableSourceLinkValidation", _options.DoSourcelinkValidation.ToString() },
-                { "PublishInstallersAndChecksums", _options.PublishInstallersAndChecksums.ToString() },
+                { "PublishInstallersAndChecksums", true },
```
Comment (riarenas) [inline src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs:258]: Does this somehow implicitly call true.ToString()?

Comment (alexperovich) [reply src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs:258]: this syntax just calls Dictionary.Add a bunch of times, I'm surprised this works.

Comment (riarenas) [reply src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs:258]: It does not. This PR doesn't build

Comment (epananth) [reply src/Microsoft.DotNet.Darc/src/Darc/Operations/AddBuildToChannelOperation.cs:258]: lol, its a bool so I passed it as is.. changing it

**Pull request merged**